airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -222,10 +238,12 @@ export const SignupForm: React.FC = () => {
     >
       {({ isValid, isSubmitting, values, status }) => (
         <Form>
-          <RowFieldItem>
-            <NameField />
-            <CompanyNameField />
-          </RowFieldItem>
+          {(!hideName || !hideCompanyName) && (
```

Comment:
Another possible way to avoid checking for nots so often. Positive can be easier to read.
```
const showName = !useExperiement("authPage.signup.hideName", false);

Reply:
Yeah it was starting to feel like too many nots, so what you suggested is probably a better approach